src/asmcnc/comms/router_machine.py

```diff
@@ -1204,14 +1204,20 @@ def g28_z(self): return float(self.s.g28_z)
 
 # SPEED AND FEED GETTERS
     def feed_rate(self): return int(self.s.feed_rate)
+
     def spindle_speed(self): 
         if self.spindle_voltage == 110: 
             # if not self.spindle_digital or not self.fw_can_operate_digital_spindle(): # this is only relevant much later on
             converted_speed = self.convert_from_230_to_110(self.s.spindle_speed)
             return int(converted_speed)
         else: 
             return int(self.s.spindle_speed)
-    def spindle_load(self): return int(self.s.spindle_load_voltage)
+
+    def spindle_load(self): 
+        try:
+            return int(self.s.spindle_load_voltage)
+        except:
+            return ''
```

Comment:
potentially unsafe? returning strict int in good case otherwise returning a string

Reply:
oooh good point... it's default value in serial is None (so I was trying to protect against it passing that back) but actually maybe that's better than an int/str situation. 

the plus side is that at the moment, it isn't actually used by anything else in the code 😅

Reply:
tempted to leave it for now until there's an actual 'use case' for it